src/axom/slam/Utilities.hpp

```diff
@@ -47,6 +48,120 @@ struct TypeToString<double>
   static std::string to_string() { return "double"; }
 };
 
+/**
+ * \brief A simple 3D point class similar to primal's point class,
+ * with some basic Point/Vector functionalities
+ */
+template <typename DataType = double>
+struct Point3
```

Comment:
Unfortunately it can't  -- `slam` doesn't depend on `primal` and this class exists to help test some slam features using only slam and core. 

The Delaunay tests use `primal::Point` as the Point template, but for internal testing, slam::IAMesh uses the slam::Point3 class.

Reply:
@kennyweiss , please add a sentence to the struct's Doxygen explaining what you just wrote.